sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/NuGetMetadataReference.cs

```diff
@@ -174,6 +181,11 @@ private static MetadataReference[] CreateReferences(string packageId, string pac
         internal static Dictionary<string, MetadataReference[]> XunitExtensibilityCore =>
             lazyXunitExtensibilityCore.Value;
 
+        internal static MetadataReference[] XunitV1Packages =>
+            lazyXunitV1.Value.Values.First()
+            .Concat(lazyXunitExtensionsV1.Value.Values.First())
+            .ToArray();
+
```

Comment:
There are now a few tests that specifically check the legacy (v1) _xUnit_ framework. There's no particular reason to test multiple versions of the legacy framework, and there's no need reason for the individual tests to have to know which version of _xUnit_ is installed, so the property exposes an array of references rather than a dictionary.